src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -99,6 +103,13 @@ private static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo
                 author = Author.UNKNOWN_AUTHOR;
             }
 
+            if (config.shouldIncludeLastModifiedDate()) {
+                // convert the commit date from the author time zone to cli-specified timezone
+                Date convertedCommitDate = TimeUtil.convertDateWithZones(new Date(commitDateInMs),
+                        ZoneId.of(authorTimeZone), ZoneId.of(config.getZoneId()));
```

Comment:
I can't remember exactly, but if `commitDateInMs` is a number that is the number of milliseconds from epoch, then we need to convert from the system time zone to the user-specified time zone, not from the author's time zone.

The milliseconds represents a point in time, which is converted to a `Date` object based on the system time zone. We then convert the date (in the system time zone) to the user-specified time zone so that when we drop the hour, minute and seconds, the year, month and day is still correct.